src/transactions/sendBtcTransaction.ts

```diff
@@ -2,7 +2,17 @@ import type { Json } from 'jsontokens';
 import { createUnsecuredToken } from 'jsontokens';
 
 import { getDefaultProvider } from '../provider';
-import type { SendBtcTransactionOptions } from './types';
+import type { Recipient, SendBtcTransactionOptions, SerializedRecipient, SerializedSendBtcTransactionPayload } from './types';
+
+const serializer = (recipient: Recipient[]): SerializedRecipient[] => {
```

Comment:
There is another simpler way to serialize BigInt:
https://github.com/secretkeylabs/wallet-connect/blob/master/webapp/src/App.js#L23C1-L25C3

Reply:
oh i see i'll update the PR then

Reply:
Makes sense, let's keep the current implementation.